phpBB/includes/ucp/ucp_pm.php

```diff
@@ -398,7 +398,7 @@ function main($id, $mode)
 				{
 					$template->assign_vars(array(
 						'S_VIEW_MESSAGE'		=> true,
-						'L_RETURN_TO_FOLDER'	=> $user->lang('RETURN_TO', $folder_status['folder_name']),
+						'L_RETURN_TO_FOLDER'	=> $user->lang('RETURN_TO', ($folder_status ? $folder_status['folder_name'] : '')),
```

Comment:
I don't think there is a need for the brackets

Reply:
Fixed.